src/commands/dynorestart.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+
+import Command from 'cli-engine-command'
+import { default as Duration, FLAG as DurationFlag } from '../mixins/duration'
+
+export default class extends Command {
+  static topic = 'cli'
+  static command = 'dynorestart'
+  static description = 'this is an example command showing duration parsing. It takes five seconds to complete.'
+  static flags = [
+    DurationFlag
+  ]
+  duration = new Duration(this)
+
+  run () {
+    console.log('Restarting dynos. This will take five seconds. That is too long')
+    return Promise.race([this.duration.wait(), this.main()])
+  }
+
+  main () {
+    return new Promise(resolve => {
```

Comment:
a way you could make this cleaner is by creating a utility function wait:

```js
const wait = msg => new Promise(resolve => setTimeout(resolve, ms))
```

Then use it here like this:

```js
await wait(5000)
console.log('Done restarting dynos')
```

You could reuse it below as well.